RELEASING.md

```diff
@@ -44,8 +44,11 @@ Publish the release notes on GitHub and ask our community manager to announce th
 
 ## Creating the final release
 
+Make sure to have the latest changes from `upstream` in your local release branch.
+
 ```
 git checkout release-vX.Y.Z
```

Comment:
I'd add a `git fetch upstream` here, to emphasize the need to be up-to-date with `upstream` (in case no `fetch`es occured since the release branch was created).

Reply:
I don't understand, isn't this exactly what `git pull` is doing in the next line?

Reply:
If there is no local `upstream/release-vX.Y.Z`, the `git checkout` command will fail, a previous `fetch` prevents that.

Reply:
Sorry, I still don't understand how this relates to my proposed steps.

Reply:
The readme says: 
>Make sure to have the latest changes from `upstream` in your local release branch.

However, if the branch was created from a different computer, the one the final release is being made from may have not performed a `fetch` since then, and will therefore not know about the `upstream/release-vX.Y.Z` branch, making `checkout` fail. A previous explicit `fetch` prevents that.

This is, granted, a somewhat contrived scenario, and I'd expect the releaser to know enough of `git` to not need this help, but I'd still add it for completeness sake.

Reply:
I prefer to leave the snippets simple even if somewhat incomplete, and later automate all of this in a script. Does that sound ok?